<?xml version="1.0"?>
<launch>
    <!-- Simulator Bridge -->
    <include file="$(find styx)/launch/server.launch" />

    <!--DBW Node -->
    <include file="$(find twist_controller)/launch/dbw_sim.launch"/>

    <!--Waypoint Loader -->
    <include file="$(find waypoint_loader)/launch/waypoint_loader.launch"/>

    <!--Waypoint Follower Node -->
    <include file="$(find waypoint_follower)/launch/pure_pursuit.launch"/>

    <!--Waypoint Updater Node -->
    <include file="$(find waypoint_updater)/launch/waypoint_updater.launch"/>

    <!--Traffic Light Detector Node -->
    <include file="$(find tl_detector)/launch/tl_detector.launch"/>

    <!-- Charting tools for debugging -->
    <include file="$(find tools)/launch/chart.launch" />

    <!--Traffic Light Locations and Camera Config -->
    <param name="traffic_light_config" textfile="$(find tl_detector)/sim_traffic_light_config.yaml" />



    <group ns="dbw_node">
        <param name="vehicle_mass" value="1080." />
        <param name="fuel_capacity" value="0." />
        <param name="brake_deadband" value=".2" />
        <param name="decel_limit" value="-5." />
        <param name="accel_limit" value="1." />
        <param name="wheel_radius" value="0.335" />
        <param name="wheel_base" value="3" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
    </group>
</launch>
